proto/ibc/applications/interchain_accounts/controller/v1/tx.proto

```diff
@@ -28,6 +28,7 @@ message MsgRegisterInterchainAccount {
 // MsgRegisterInterchainAccountResponse defines the response for Msg/RegisterAccount
 message MsgRegisterInterchainAccountResponse {
   string channel_id = 1 [(gogoproto.moretags) = "yaml:\"channel_id\""];
+  string port_id = 2 [(gogoproto.moretags) = "yaml:\"port_id\""];
```

Comment:
Could you run `make proto-all` or `make proto-format`? Looks like the alignment is off, maybe you just used the `proto-gen` target.

Reply:
Hi sir @damiannolan , thanks for your reviewed. I've fixed it